Application/Analyzing_Application_Issues.md

```diff
@@ -0,0 +1,130 @@
+# Analyzing Application issues for services deployed to an Azure Service Fabric Cluster
+
+Applications deployed to an Azure Service Fabric cluster can face issues due to various reasons and in most cases Application code or config can cause these issues. Since the Application team understands the architecture of the application and its dependencies, they are in the best position to quickly determine cause and resolve it. Sometimes (not always) Service Fabric CSS team can provide insights from Service Fabric logs to speedup investigation by Application team. During these interactions between the customer team and SF CSS team for Application issues, it is important that Application team engineers are engaged directly in the communications, and they provide requested information in a timely manner. It is also important for application team to provide details of their investigations, as early as possible ideally when opening the case.
+
+While SF CSS team can help with investigations, the final responsibility of analyzing and determining the root cause of Application issues is the Application team. If the Application team after their analysis suspect some behavior of Service Fabric to be the cause for Application issue, application team should provide details of their investigation to help speed up investigation by SF CSS team.
+
+### Common causes
+
+1. Failing on startup
+    1. [Diagnose common code package errors by using Service Fabric](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-diagnostics-code-package-errors)
+    2. Exhausting load on startup, for example loading configuration data from database or previous state
+    1. Exceeding hardware resources
```

Comment:
>1. [](http://example.com/codeflow?start=4&length=2)

(nitpick) it's all '1's (and one 2)

Reply:
Helpful, will change the text to make it clearer. Need to use familiar wording and provide examples, so that customers can find themselves in it.